docker/base-deps/Dockerfile

```diff
@@ -48,16 +46,22 @@ RUN sudo apt-get update -y && sudo apt-get upgrade -y \
         netbase \
         openssh-client \
         gnupg; fi) \
-    && wget \
-        --quiet "https://repo.anaconda.com/miniconda/Miniconda3-py37_23.1.0-1-Linux-${HOSTTYPE}.sh" \
+    && if [ "$PYTHON_VERSION" = "3.11" ]; then \
+            MINICONDA_VERSION="py310_23.3.1-0"; \
+            LIBGCC="libgcc-ng"; \
+        else \
+            MINICONDA_VERSION="py37_23.1.0-1"; \
+            # TODO (can): Remove libffi=3.3 once #33299 is resolved
+            LIBGCC="libgcc-ng libffi=3.3"; \
```

Comment:
and `LIBGCC=(libgcc-ng libffi=3.3)` here.

Reply:
Lol I tried this and [[ ]], both are rejected by Docker for some reason. E.g. this is the error for using (): https://buildkite.com/ray-project/oss-ci-build-pr/builds/26300#0188e022-f8ab-4847-b721-db4c8cdf19ad. 

Going back to using [] and string seems to be ok.

Maybe I poke you tomorrow on the syntax to see what's wrong